client/src/components/History/Multiple/MultipleView.vue

```diff
@@ -16,48 +16,40 @@ const filter = ref("");
 library.add(faTimes);
 
 const { currentUser } = storeToRefs(useUserStore());
+const { histories, historiesLoading, currentHistory } = storeToRefs(useHistoryStore());
 
 function updateFilter(newFilter: string) {
     filter.value = newFilter;
 }
 </script>
 
 <template>
-    <UserHistories
-        v-if="currentUser"
-        v-slot="{ histories, handlers, historiesLoading, currentHistory }"
-        :user="currentUser">
-        <div>
-            <b-alert v-if="historiesLoading" class="m-2" variant="info" show>
-                <LoadingSpan message="Loading Histories" />
-            </b-alert>
-            <div v-else-if="histories.length" class="multi-history-panel d-flex flex-column h-100">
-                <b-input-group class="w-100">
-                    <DebouncedInput v-slot="{ value, input }" v-model="filter">
-                        <b-form-input
-                            size="sm"
-                            :class="filter && 'font-weight-bold'"
-                            :value="value"
-                            :placeholder="'search datasets in selected histories' | l"
-                            data-description="filter text input"
-                            @input="input"
-                            @keyup.esc="updateFilter('')" />
-                    </DebouncedInput>
-                    <b-input-group-append>
-                        <b-button size="sm" data-description="show deleted filter toggle" @click="updateFilter('')">
-                            <FontAwesomeIcon icon="fa-times" />
-                        </b-button>
-                    </b-input-group-append>
-                </b-input-group>
-                <multiple-view-list
-                    :histories="histories"
-                    :filter="filter"
-                    :current-history="currentHistory"
-                    :handlers="handlers" />
-            </div>
-            <b-alert v-else class="m-2" variant="danger" show>
-                <span v-localize class="font-weight-bold">No History found.</span>
-            </b-alert>
+    <div v-if="currentUser">
+        <b-alert v-if="historiesLoading" class="m-2" variant="info" show>
+            <LoadingSpan message="Loading Histories" />
+        </b-alert>
+        <div v-else-if="histories.length" class="multi-history-panel d-flex flex-column h-100">
+            <b-input-group class="w-100">
+                <DebouncedInput v-slot="{ value, input }" v-model="filter">
+                    <b-form-input
+                        size="sm"
+                        :class="filter && 'font-weight-bold'"
+                        :value="value"
+                        :placeholder="'search datasets in selected histories' | l"
+                        data-description="filter text input"
+                        @input="input"
+                        @keyup.esc="updateFilter('')" />
+                </DebouncedInput>
```

Comment:
```suggestion
                <b-form-input
                    size="sm"
                    v-model="filter"
                    debounce="500"
                    :class="filter && 'font-weight-bold'"
                    :placeholder="localize('search datasets in selected histories')"
                    data-description="filter text input"
                    @keyup.esc="updateFilter('')" />
```

Fixes the typescript errors